simulation/src/main/kotlin/it/unibo/collektive/examples/spreading/MaxNetworkId.kt

```diff
@@ -0,0 +1,9 @@
+package it.unibo.collektive.examples.spreading
+
+import it.unibo.collektive.aggregate.api.Aggregate
+import it.unibo.collektive.alchemist.device.sensors.EnvironmentVariables
+import it.unibo.collektive.stdlib.spreading.gossipMax
+
+/** Identify the maximum ID values in the network. */
+fun Aggregate<Int>.maxNetworkID(environment: EnvironmentVariables): Int =
+    gossipMax(localId).also { maxValue -> environment["isMaxID"] = localId == maxValue }
```

Comment:
Be consistent in using `it` in lambdas:

```suggestion
    gossipMax(localId).also { environment["isMaxID"] = localId == it }
```